facefusion/processors/frame/modules/frame_enhancer.py

```diff
@@ -26,23 +25,13 @@
 NAME = __name__.upper()
 MODELS : ModelSet =\
 {
-	'real_esrgan_x2plus':
-	{
-		'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models/real_esrgan_x2plus.pth',
-		'path': resolve_relative_path('../.assets/models/real_esrgan_x2plus.pth'),
-		'scale': 2
-	},
-	'real_esrgan_x4plus':
-	{
-		'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models/real_esrgan_x4plus.pth',
-		'path': resolve_relative_path('../.assets/models/real_esrgan_x4plus.pth'),
-		'scale': 4
-	},
-	'real_esrnet_x4plus':
+	'hfa2k_x4':
```

Comment:
Add every model from that repository, helps me to save time on my side (kinda busy)

Plan: a pool to let the user make a decision

Reply:
what should be default? ''hfa2k_x4''?

Reply:
yes

Reply:
i removed this model. it is listed as anime model.